### ISSUE TITLE
Can't get the Plugin to Work!

### ISSUE THREAD

**Issue by fabinhorsff** [ksqsf/logseq-open-in-emacs#3]
Hello!

I have configured Emacs to open Org-Protocol. 
The verification (emacsclient -c 'org-protocol://find-file?path=/tmp/some_file.txt') correctly opens a new frame in Emacs.

But when asking Logseq to edit the page, it does not open a new window.
I am on a Mac OS running 10.15.7 (Catalina).
Emacs 28.2 with Doom Emacs (don't think this is relevant. 
Logseq 0.8.18.

Any light on how to make logseq call Emacs?
Thanks!

**Comment by ksqsf** [comment]
Hi, two things come to my mind:

1. Can you confirm that the command `emacsclient -c 'org-protocol://find-file?path=/tmp/some_file.txt'` opens the file `/tmp/some_file.txt`?

   If you didn't load `org-protocol` properly, then `emacsclient` can also open a frame, but does not visit that particular file.

2. Can you confirm that you have registered the org-protocol protocol in your system?  For example, use a Web browser to access `org-protocol://find-file?path=/tmp/some_file.txt` and see if Emacs is invoked to open `/tmp/some_file.txt`.

   If you are using MacOS, check out [this guide](https://github.com/xuchunyang/setup-org-protocol-on-mac).  (`EmacsClient.app` recommended on the official guide doesn't work, unfortunately.)

**Comment by fabinhorsff** [comment]
Hello!

1) I can confirm that the emacsclient command issued on the command line does indeed open the file requested correctly. 

2) I did try to register the protocol in the web-browser, but could not get it to work. I don´t know if this issue is with org-protocol, or with the browser config.

3) I did follow the MacOS Guide  https://github.com/xuchunyang/setup-org-protocol-on-mac. In the issues section, someone suggests that this guide is out-dated and that it is better to follow the original guide here: https://www.orgroam.com/manual.html#Mac-OS.  This second guide seems more complete in its instructions. 

One this second guide, there is a detail related to where the put the keys in info.plist. I will try later!

Thanks for the response!  I´m still working on this and might have more information later!
Fabio
